tools/Az.Tools.Installer/help/Install-AzModule.md

```diff
@@ -37,7 +37,7 @@ Install-AzModule -Repository PSGallery
 
 ### EXAMPLE 2
 ```
-Install-AzModule Storage,Compute,Network,Blockchain -Repository PSGallery -AllowPrerelease
+Install-AzModule Storage,Compute,Network -Repository PSGallery -AllowPrerelease
```

Comment:
@msJinLei hey, since Blockchain is retiring I removed it from an example of Az.Tools.Installer, please review. Thanks.